### ISSUE TITLE
crash in appcenter distribute (xamarin) reported by appcenter diagnostic ;)

### ISSUE THREAD

**Issue by softlion** [Microsoft/AppCenter-SDK-DotNet#717]
Hi,
just got a crash reported by appcenter diagnostic :)

The crash happened in appcenter distribute for Android (xamarin).

Hope you'll find it.

### **Description**

```
java.lang.NullPointerException: Attempt to invoke interface method 'int android.content.SharedPreferences.getInt(java.lang.String, int)' on a null object reference
com.microsoft.appcenter.utils.storage.StorageHelper$PreferencesStorage.getInt(StorageHelper.java:162)
com.microsoft.appcenter.distribute.DistributeUtils.getStoredDownloadState(DistributeUtils.java:98)
com.microsoft.appcenter.distribute.Distribute.onActivityCreated(Distribute.java:452)
android.app.Application.dispatchActivityCreated(Application.java:197)
android.app.Activity.onCreate(Activity.java:966)
```
### **Repro Steps**

Dunno ;)

**Comment by guperrot** [comment]
Hi, thanks for the crash report.
It should be fixed for the next release end of this month.

**Comment by guperrot** [comment]
Hi, this bug fix is released into version [1.9.0](https://github.com/Microsoft/AppCenter-SDK-DotNet/releases/tag/1.9.0).
